lib/spotlight/dor/resources/version.rb

```diff
@@ -2,7 +2,7 @@ module Spotlight
   module Dor
     # :nodoc:
     module Resources
-      VERSION = '0.4.1'
+      VERSION = '0.5.0'
```

Comment:
I was going to bump to 0.5.0  for my recent merge with the indexing changes ...   should I wait for this one now?